Sources/scipio/Options.swift

```diff
@@ -29,6 +29,11 @@ struct BuildOptionGroup: ParsableArguments {
           help: "Whether generated frameworks are Static Frameworks or not")
     var shouldBuildStaticFramework = false
 
+    @Flag(name: [.customLong("library-evolution")],
+          inversion: .prefixedEnableDisable,
```

Comment:
Thanks to this option, swift-argument-parser automatically generates two flags `enable-library-evolution` and `disable-library-evolution`.